app/src/main/java/com/example/androidacademyhomework/UI/MovieDetailsFragment.kt

```diff
@@ -13,7 +15,18 @@ class MovieDetailsFragment : Fragment() {
         inflater: LayoutInflater, container: ViewGroup?,
         savedInstanceState: Bundle?
     ): View? {
-
         return inflater.inflate(R.layout.fragment_movie_details, container, false)
     }
+
+    override fun onViewCreated(view: View, savedInstanceState: Bundle?) {
+        super.onViewCreated(view, savedInstanceState)
+
+        view.findViewById<Button>(R.id.buttonViewBack).setOnClickListener {
+            returnToMovieListFragment()
+        }
+    }
+
+    private fun returnToMovieListFragment() {
+       findNavController().navigate( R.id.action_movieDetailsFragment_to_movieListFragment)
```

Comment:
Does it have just pop back?

Reply:
Yes, it does. 
In action 
app:popUpTo="@id/movieListFragment"
app:popUpToInclusive="true"

Reply:
No, it doesn`t.
 In action, it uses 
app:popUpTo="@id/movieListFragment"
 app:popUpToInclusive="true"